python/nano/src/bigdl/nano/quantization/neural_compressor/pytorch/utils/dataloader.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import torch
+
+
+def _check_data_type(data):
+    if isinstance(data, torch.Tensor):
+        return
+    else:
+        for x in data:
+            assert isinstance(x, torch.Tensor), ValueError
+
+
+def _check_loader(loader):
+    if loader is None:
+        return
+    sample = next(iter(loader))
+    try:
+        x, y = sample
```

Comment:
maybe add a comment here, `# if len(sample)>2, a ValueError will be raised`

Reply:
accommodate checking on `len(sample) !=2` in new version.